src/inmanta/agent/write_barier_executor.py

```diff
@@ -21,10 +21,10 @@
 import uuid
 from copy import deepcopy
 
-import inmanta.types
+import inmanta.types  # noqa: F401
```

Comment:
Not exactly sure if this is needed or we can remove it

Reply:
it not used, remove